packages/react-components/src/atoms/Avatar.tsx

```diff
@@ -82,6 +82,7 @@ const initialsStyle = css({
 const textStyle = css(fontStyles, headlineStyles[3], {
   dominantBaseline: 'central',
   textAnchor: 'middle',
+  fontSize: '1.42em',
```

Comment:
strange number, does this come directly from James? Seems roughly similar to our h3 font size that's the only similar thing I can think of

Reply:
it's odd... James asked it to be changed to `17pt` :/

Reply:
```suggestion
  fontSize: `${17/perRem}em`,
```

Reply:
That's `1em` though, so does nothing

Reply:
That's not what we want. This is not a conversion from PX to EM, but rather PT to EM.

Reply:
If you try changing e.g. ticking and unticking border in the Avatar story, you see that the text also scales because the circle becomes a bit smaller and larger

Reply:
Perhaps hop onto a quick call with James explaining this and trying to find a base size that works well in any scaling?

Reply:
The `Avatar` layouting is very intricate (I was the last to significantly touch this a couple of months ago). This is because the `Avatar` can scale to various sizes between its min and max depending on how much space the container gives it, plus the `border` can modify it, ...
The placeholder text really being an SVG that is handled the same way as a real avatar image is the only thing that makes it somewhat manageable, so changing this to behave like normal text would be a major effort in re-doing the layout and I'm not even sure it's possible with pure CSS to make a placeholder avatar always behave like the real image avatar.
So yeah if we explain to James that it scales relative to the Avatar size and settle on a size taking that into account that would be great.

Reply:
I guess James only complaint is the text size on the dashboard page, where the avatar is shown with the smallest size (I guess). The proportion looks better with a smaller text, just the oddity of its size. I've already asked him to check if its possible to get us a more "normal" value for the font size.

Can I merge this and proceed and come back to it later?

Reply:
If it's just about making it bigger for now what about 1.5? Something as specific as this will leave someone at some point in the future wondering why it needs to be this exact value and being afraid to touch it 😅